Source/TutorialModel.swift

```diff
@@ -0,0 +1,92 @@
+import UIKit
+
+@objc public class TutorialModel: NSObject {
+
+  // MARK: Public methods
+
+  public var image: UIImage? {
+    get {
+      return imageView.image
+    }
+    set(newImage) {
+      imageView.image = newImage
+    }
+  }
+
+  public var title: String? {
+    get {
+      return titleLabel.text
+    }
+    set(newValue) {
+      titleLabel.text = newValue
+    }
+  }
+  public var text: String? {
+    get {
+      return textLabel.text
+    }
+    set(newText) {
+      textLabel.text = newText
+    }
+  }
+
+  lazy private(set) var imageView: UIImageView = {
+    let imageView = UIImageView()
+    imageView.autoresizingMask = .FlexibleLeftMargin | .FlexibleRightMargin
+
+    return imageView
+    }()
+
+  lazy private(set) var titleLabel: UILabel = {
+    let label = UILabel(frame: CGRectNull)
+
+    label.autoresizingMask = .FlexibleLeftMargin | .FlexibleRightMargin
+    label.font = UIFont(name: "DIN Alternate", size: 48.0)
```

Comment:
Kind of weird that we supply a font for the default things.

We could easily drop font and textColor here and make it a requirement, same as Form.